django/utils/crypto.py

```diff
@@ -17,7 +18,11 @@
 
 # Use the system PRNG if possible
 try:
-    random = random.SystemRandom()
+    if sys.version_info >= (3, 6):
+        import secrets
+        random = secrets.SystemRandom()
+    else:
+        random = random.SystemRandom()
```

Comment:
I'm not sure what Aymeric had in mind but I see that `secrets.py` does `from random import SystemRandom` so as far as I can tell, this doesn't change any behavior or add security.